API/src/app.js

```diff
@@ -0,0 +1,32 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+
+import orders from './routes/ordersRoute';
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use('/api/v1', orders);
+
+app.use((request, response, next) => {
+  const error = new Error('unknown url path');
+  error.status = 404;
+  return next(error);
+});
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
I don't think it's necessary to disable the rules on this line. You should be able to remove the arguments that are not used.

Reply:
If I remove the `next` argument, the middleware will not work. I wanted to create a middleware that catches and resolves all errors.

Reply:
You aren't using next though. I don't understand what you mean by it wouldn't work.

Reply:
It works but when I pass an error object into `next` in any of the middlewares, I wanted this error handler alone to catch it. That syntax is the convention for error handlers. Without `next`, the default error handler will throw an error on the console.
Please check this: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/Introduction#Handling_errors
and this: https://github.com/expressjs/express/issues/2896

Reply:
Why not return the response with the error within the middle-ware that the error is found?